docs/baking.md

```diff
@@ -52,35 +59,33 @@ sudo add-apt-repository 'deb http://ppa.launchpad.net/serokell/tezos/ubuntu bion
 sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 37B8819B7D0D183812DCA9A8CE5A4D8933AE7CBB
 sudo apt-get update
 # Install packages
-sudo apt-get install tezos-client tezos-node tezos-baker-<proto> tezos-endorser-<proto>
+sudo apt-get install tezos-baking
 ```
 
 Packages for `tezos-node`, `tezos-baker-<proto>` and `tezos-endorser-<proto>` provide
-systemd units for running the corresponding binaries in the background.
-
-## Setting up the tezos-node
-
-The easiest way to set up a node running on the `mainnet` or one of the
-testnets (e.g. `edo2net`) is to use one of the predefined
-`tezos-node-<network>.services` systemd services provided in the `tezos-node`
-package.
+systemd units for running the corresponding binaries in the background, these units
+are orchestrated by the services provided by the `tezos-baking` package.
```

Comment:
seems kinda repetitive mentioning the `tezos-baking` package here, maybe instead say:
```suggestion
are orchestrated by the `tezos-baking-<network>` units.
```